src/Model/File.php

```diff
@@ -30,15 +30,18 @@ class File extends Model
     /** @var Filesystem */
     public $flysystem;
 
-    /** @bool Should we automatically create thumbnail for images */
+    /** Should we automatically create thumbnail for images */
     public bool $createThumbnail = true;
 
-    /** @int Thumbnail image max width in pixels */
+    /** Thumbnail image max width in pixels */
     protected int $thumbnailMaxWidth = 150;
 
-    /** @int Thumbnail image max height in pixels */
+    /** Thumbnail image max height in pixels */
     protected int $thumbnailMaxHeight = 150;
 
+    /** Thumbnail format - \IMAGETYPE_PNG, \IMAGETYPE_JPEG and \IMAGETYPE_GIF are supported */
+    protected int $thumbnailFormat = \IMAGETYPE_PNG;
```

Comment:
```suggestion
    /** @var 'png'|'jpg'|'gif' Thumbnail format */
    protected string $thumbnailFormat = 'png';
```

to be library/GD indepent